src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -603,19 +793,31 @@ def compute_metadata(
         # Metadata
         metadata: Dict[str, MetadataType] = {}
 
-        # Orchestrator Run ID
-        if run_id := self._compute_orchestrator_run_id(execution):
-            metadata[METADATA_ORCHESTRATOR_RUN_ID] = run_id
+        # Handle schedule metadata if execution is a dict
+        if isinstance(execution, dict):
+            metadata.update(
+                {
+                    "schedule_rule_name": execution["rule_name"],
+                    "schedule_type": execution["schedule_type"],
+                    "schedule_expression": execution["schedule_expr"],
+                    "pipeline_name": execution["pipeline_name"],
+                }
+            )
+
+            if next_execution := execution.get("next_execution"):
+                metadata["next_execution_time"] = next_execution.isoformat()
```

Comment:
Question to @schustmi and @bcdurak , i made this so that some metadata is captured in the schedule case, but not all. Can you help me fix this? Its probably flawed logic as it doesn't show up on the dashboard at all